src/main/java/at/hannibal2/skyhanni/features/commands/WikiManager.kt

```diff
@@ -9,21 +9,20 @@ import at.hannibal2.skyhanni.utils.ItemUtils.nameWithEnchantment
 import at.hannibal2.skyhanni.utils.KeyboardManager.isKeyHeld
 import at.hannibal2.skyhanni.utils.LorenzUtils
 import at.hannibal2.skyhanni.utils.NEUItems
-import at.hannibal2.skyhanni.utils.OSUtils
 import at.hannibal2.skyhanni.utils.StringUtils.removeColor
 import net.minecraft.client.gui.inventory.GuiContainer
 import net.minecraft.item.ItemStack
 import net.minecraftforge.client.event.GuiScreenEvent
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.net.URLEncoder
 
-class WikiManager {
+object WikiManager {
+    private const val OFFICIALURLPREFIX = "https://wiki.hypixel.net/"
+    private const val OFFICIALSEARCHPREFIX = "index.php?search="
+    private const val FANDOMURLPREFIX = "https://hypixel-skyblock.fandom.com/wiki/"
+    private const val FANDOMSEARCHPREFIX = "Special:Search?query="
```

Comment:
Also should these be consts and be all caps?
Intellij told me to do this

Reply:
Constants should be upper case, this is correct. Just please add an underline character between words: THIS_WAY